emscripten.py

```diff
@@ -55,9 +55,8 @@ def compute_minimal_runtime_initializer_and_exports(post, exports, receiving):
 
 
 def write_output_file(outfile, module):
-  for i in range(len(module)): # do this loop carefully to save memory
```

Comment:
The `# do this loop carefully to save memory` part is no longer needed for two reasons.

1. We are no longer generating garbage inside the loop
2. We are not processing O(program size) inputs here like perhaps we did back in the fastcomp days.  The inputs here are just the size of the JS library support code.